File: branches/simosws/simosws/BusinessLayer/DTO/EncuestaSenderObject.vb
Public Class EncuestaSenderObject

    Private _encuesta As Encuestado
    Public Property encuesta() As Encuestado
        Get
            Return _encuesta
        End Get
        Set(ByVal value As Encuestado)
            _encuesta = value
        End Set
    End Property


    Private _respuestas As List(Of Respuesta)
    Public Property respuestas() As List(Of Respuesta)
        Get
            Return _respuestas
        End Get
        Set(ByVal value As List(Of Respuesta))
            _respuestas = value
        End Set
    End Property



End Class

File: branches/simos/src/gob/sis/simos/dto/EncuestaSenderObject.java
package gob.sis.simos.dto;

import java.util.List;

import com.google.gson.Gson;
import com.google.gson.annotations.Expose;

import gob.sis.simos.entity.Encuesta01;
import gob.sis.simos.entity.Respuesta;

public class EncuestaSenderObject {

	@Expose
	private Encuesta01 encuesta;
	@Expose
	private List<Respuesta> respuestas;
	
	public Encuesta01 getEncuesta() {
		return encuesta;
	}
	public void setEncuesta(Encuesta01 encuesta) {
		this.encuesta = encuesta;
	}
	public List<Respuesta> getRespuestas() {
		return respuestas;
	}
	public void setRespuestas(List<Respuesta> respuestas) {
		this.respuestas = respuestas;
	}
	
	public String toJson(Gson gson){
		return gson.toJson(this);
	}
	
}
